### PR TITLE
fix(studio): update to grpo hyperparameters

### PR DESCRIPTION
**Pull request by nakolean**
https://github.com/user-attachments/assets/47b589a1-81e6-4927-8e90-12ae53399508

## Type of Change

- [ ] Code change (feature, bug fix, or refactor)
- [ ] Code change with documentation updates
- [ ] Documentation only
- [ ] Contributor tooling or automation
- [ ] CI, build, or test infrastructure

## Quality Gates

- [ ] Tests added or updated for changed behavior
- [ ] Existing tests cover changed behavior — justification:
- [ ] Tests not applicable — justification:
- [ ] Documentation updated for user-visible behavior
- [ ] Documentation not applicable — justification:

## Verification

- [ ] Pull request title follows the repository's Conventional Commit format
- [ ] Every commit includes an appropriate `Signed-off-by:` trailer
- [ ] `uv run pre-commit run -a` passes, or any blocked checks are identified below
- [ ] Targeted tests pass, or tests are marked not applicable above
- [ ] No secrets, API keys, or credentials are included

Targeted validation:

## Summary by CodeRabbit

* **Improvements**
  * Updated GRPO controls with clearer guidance, revised defaults, and practical limits for learning rate, batching, validation, clipping, and optimizer settings.
  * Rollout groups now require at least two generations, with improved rollout sizing and batch-divisibility checks.
  * Added validation for reward environments, sequence lengths, and supported LoRA configurations.
  * GRPO validation intervals now use training steps and are bounded by run length.
  * Switching between DPO and GRPO resets method-specific settings while preserving shared options.
  * Clarified Triton LoRA behavior under tensor parallelism.

### CODE REVIEW

File: web/packages/studio/src/util/forms/customization.ts [hunk 1]
```diff
@@ -191,6 +191,15 @@ export const RL_GRPO_TRAINING_DEFAULTS = {
   // Backend default is 0.0 for GRPO and 0.05 for DPO; the 0.1 inherited from
   // RL_DPO_DEFAULTS would otherwise apply a KL penalty the user never asked for.
   ref_policy_kl_penalty: 0.0,
+  // The shared 1e-4 is SFT-scale and collapses a policy within a few dozen RL steps;
+  // the platform GRPO fixtures train at 5e-6 full-weight, 1e-5 for LoRA.
+  learning_rate: 5e-6,
```
Comment (steramae-nvidia) [inline web/packages/studio/src/util/forms/customization.ts:231]: Does this mean you need a conditional to set this to 1e-5?

**Review by steramae-nvidia — APPROVED** [review]

File: web/packages/studio/src/util/forms/customization.ts (hunk 1 above)
Comment (nakolean) [reply web/packages/studio/src/util/forms/customization.ts:231]: We don't need to, updated the comment to avoid confusion in the future.

**Review by htolentino-nvidia — APPROVED** [review]
